lunabot_control/scripts/manual_controller.py

```diff
@@ -93,6 +98,7 @@ def __init__(self):
 
         self.driving_mode = "Forwards"
         
+        self._max_speed = rospy.get_param("~max_speed", 2.0) * 60 / 2 / np.pi # In rad/s converted to RPM
```

Comment:
Same thing that applies to the other max_speed thing in other file. Should be defined as RPM explicitly.